pubnet/horizon/etc/stellar-captive-core.cfg

```diff
@@ -0,0 +1,203 @@
+# captive core config for futurenet
+NETWORK_PASSPHRASE="__NETWORK__"
+# disable the web service port, not used
+HTTP_PORT=0
+PUBLIC_HTTP_PORT=false
+# To avoid conflicts with the core instance
+PEER_PORT=11725
+DATABASE="__DATABASE__"
```

Comment:
it may be worth enabling BUCKETLISTDB, it may help performance of captive core when it's using on-disk db, it's being done on rpc captive core config for example - https://github.com/stellar/quickstart/blob/horizon-captive/futurenet/soroban-rpc/etc/stellar-captive-core.cfg#L11-L12

Reply:
Opened issue to track doing this across the board in the image:
- https://github.com/stellar/quickstart/issues/490#issue-1897318064

I'll keep the cfg as it is in this PR, as it's just copying what's already being done for horizon in other networks. After this merges I'll follow up with #490.